pyproject.toml

```diff
@@ -94,6 +94,8 @@ develop = [
 ]
 examples = [
   "salesforce-merlion<2.1",
+  "pycaret[analysis,models,tuner,parallel,test]==3.1.0",
```

Comment:
It looks like the build fails on installing NumPy [^1]. Is it introduced by `pycaret` and/or one of its extras? I guess so, but why does it fail? Do we need to limit the upper bound?

[^1]: https://github.com/crate-workbench/mlflow-cratedb/actions/runs/6691387798/job/18178515463?pr=45#step:5:1616

Reply:
Oh. The build is selecting `numpy==1.19.3`. Why?

-- https://github.com/crate-workbench/mlflow-cratedb/actions/runs/6691387798/job/18178515463?pr=45#step:5:567

Reply:
Ah ok we have 3.11 in this project - this might be an issue.
I'm still bound to 3.10 as a lot of data science tools still don't support 3.11.
Let me check the dependency tree very quick. Maybe we can work around this issue.

Reply:
Ok it's confirmed, we can't have pycaret on python 3.11.
[PR](https://github.com/pycaret/pycaret/pull/3756) aims to change this. 
I'll change this PR to Draft for now and update, once we have Python 3.11 support.
I guess the example itself is still useful for people wanting to use automl with cratedb for now

Reply:
Thank you for referring to https://github.com/pycaret/pycaret/pull/3756. Not supporting Python 3.11 or pandas 2.x is sad, but c'est la vie. Depending on progress there, or capacity on our end, we may downgrade CI to Python 3.10 in order to bring this in.

Reply:
GH-48 will downgrade to Python 3.10.

Reply:
Hi again,

CI signals success now, but it looks like the branch has not been correctly rebased on top of the `main` branch. Let me know if you want me to support you by taking over.

With kind regards,
Andreas.